google/resource_composer_environment_test.go

```diff
@@ -0,0 +1,506 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"google.golang.org/api/composer/v1"
+	"log"
+	"strings"
+)
+
+const environmentPrefix = "tf-composer-testenv"
```

Comment:
nit: these are scoped package-wide, so it's probably best to name it something like composerEnvironmentPrefix